buildsystem/mavenize/test.properties/el-testjee.wildfly.properties

```diff
@@ -0,0 +1,53 @@
+#JEE server properties - WildFly
+#Properties used there are used by cargo-maven2-plugin. For more details see cargo-maven2-plugin documentation page.
+#Server family (value is specified by cargo-maven2-plugin).
+server.name=jboss
+
+#Server type/major version (value is specified by cargo-maven2-plugin).
+server.containerId=wildfly15x
+
+#Hostname to which server binds. Used by testing client (EJB client) too.
+server.host=localhost
+
+#Server HTTP port. Web console is usually available there
+server.http.port=8080
+
+#Server RMI port.  Used by testing client (EJB client) too.
+server.rmi.port=8080
+
+#RMI protocol. Used by EJB client.
+server.rmi.protocol=http-remoting
+
+#Username used by testing client for authentication. Usually it's default server username. See cargo-maven2-plugin documentation page.
+server.usr=admin
+
+#Password used by testing client for authentication. Usually it's default server password. See cargo-maven2-plugin documentation page.
+server.pwd=adminadmin
+
+#JEE/EJB client initial context factory (driver). See RMI/EJB client JEE server documentation.
+server.initialCtxFactory=org.wildfly.naming.client.WildFlyInitialContextFactory
+
+#EclipseLink specific property/hint to identify server family.
+server.platform=JBoss
+server.platform.class=jboss-platform
+
+#JEE server client library. There are Maven artifact properties. Use Maven Central to find right values.
+jee.client.groupId=org.wildfly
+jee.client.artifactId=wildfly-ejb-client-bom
+#In case of WildFly this property is used as a substitution property in eclipselink-wildfly-module.xsl file.
+jee.client.version=15.0.1.Final
+
+#Server binaries location.
+#Just binaries are needed. cargo-maven2-plugin during maven build/tests configures server/server domain automatically in .../target/cargo directory.
+cargo.container.installation.home=/usr/local/javaApplications/wildfly-15.0.1.Final
```

Comment:
this should be changed to some temop dir - can be ie java.io.tmpdir, or maybe better sth like ${user.home}/.eclipselinktests

Reply:
By default now is WildFly automatically downloaded by _download-maven-plugin_ (see _pom_test_server.xml_). Download related properties _installation.url_, _installation.checksum.md5_ are in parent pom in wildfly, glassfish profiles.
To enable/disable it see _el-testjee.wildfly.properties_ file (_skip.jee.server.installation_, _cargo.container.installation.home_ properties).